helix-view/src/editor.rs

```diff
@@ -307,7 +307,7 @@ pub struct Editor {
 
     pub syn_loader: Arc<syntax::Loader>,
     pub theme_loader: Arc<theme::Loader>,
-
+    pub last_theme: Option<String>,
```

Comment:
I think this will easily pollute stuff later, probably need to comment that this is only used for `:theme` preview. Later I think we probably need to separate each out.